docs/user/reference/environment-variables.rst

```diff
@@ -56,6 +56,31 @@ All :doc:`build processes </builds>` have the following environment variables au
     :Example: ``https://docs.readthedocs.io/ja/stable/``
     :Example: ``https://example--17.org.readthedocs.build/fr/17/``
 
+.. envvar:: READTHEDOCS_REPOSITORY_CLONE_URL
+
+    URL for the remote source repository, from which the documentation is cloned.
+    It could be HTTPS, SSH or any other URL scheme supported by Git.
+    This is the same URL defined in your Project's :term:`dashboard` in :menuselection:`Admin --> Settings --> Repository URL`.
+
+    :Example: ``https://github.com/readthedocs/readthedocs.org``
+    :Example: ``git@github.com:readthedocs/readthedocs.org.git``
+
+.. envvar:: READTHEDOCS_REPOSITORY_IDENTIFIER
```

Comment:
Is this a repository id? Or a version ID?

Reply:
I like `READTHEDOCS_GIT_IDENTIFIER` better. In fact I like everything better with git :)

`READTHEDOCS_GIT_CLONE_URL`
`READTHEDOCS_GIT_COMMIT_HASH`
`READTHEDOCS_GIT_IDENTIFIER`

it's easier to understand. I think the old VCS semi-supported backends can just have to deal with that. If we want to really support something, we might as well re-introduce "VCS" or have stuff like `READTHEDOCS_HG_CLONE_URL`... but that's not relevant now.

Reply:
Yea, if we wanted it to be generic, we could use VCS, but I agree GIT is probably better, and forward looking 👍

Reply:
OK, I'm going to change the names to use `GIT` 👍🏼